lib/mocha_runner.coffee

```diff
@@ -0,0 +1,19 @@
+MochaArgumentBuilder = require './mocha_argument_builder'
+mochaExecutablePath = require.resolve 'mocha/bin/mocha'
+{spawn} = require 'child_process'
```

Comment:
Consider using https://www.npmjs.com/package/cross-spawn here so that this runs on Windows as well. This is open source.